components/Result.tsx

```diff
@@ -1,9 +1,33 @@
 import { Album } from "@/app/utils/discogs.schemas";
+import Image from "next/image";
+import Icon from "./Icon";
 
 export default function Result({ album }: { album: Album }) {
   return (
-    <>
-      <h3>Year: {album.year}</h3>
-    </>
+    album.title && (
+      <div className="flex p-24 border-bottom-solid border-b-2 border-stone-900">
+        {album.cover_image.endsWith(".gif") ? (
+          <div className="w-[200px] h-[200px] p-12 rounded-sm bg-gray-800">
+            <Icon
+              className="w-full h-full text-white "
+              type="no-img"
+              size="medium"
+            />
+          </div>
+        ) : (
+          <Image
+            className="rounded-sm"
+            src={album.cover_image}
+            alt={`${album.title} cover image`}
+            width={200}
+            height={200}
+          />
+        )}
+        <div className="ml-14">
+          <h3 className="text-lg font-bold">Title: {album.title}</h3>
+          <h3>Year: {album.year ? album.year : "-"}</h3>
```

Comment:
also probably don't need to include a year label since it's implied, your choice though

```suggestion
          <h3>{album.year ? album.year : "-"}</h3>
```

Reply:
Agreed, without the title is a lot cleaner